modules/@apostrophecms/asset/index.js

```diff
@@ -941,6 +943,28 @@ module.exports = {
       // Open the implementation for more dev comments.
       transformRebundledFor,
 
+      // Optional functions passed to webpack's mergeWithCustomize, allowing
```

Comment:
We're changing the default behavior of webpack merge here, so document what our new "default" behavior is. I see your comment but it's not clear to me why this behavior is more "reasonable" than the standard default.

Reply:
If it's a question of preventing duplicates then please document who "wins" — later settings become overrides? In what way?

Reply:
There is no change in the default behaviour. 
`['value', 'another', 'value']` becomes `['value', 'another']`. This is applied by default on extensions strings and resolve paths (arrays), no change in the order. The win is correct array values and in the case of the resolve paths, probably better performance. 
The developer may opt-in for change order of merging arrays as described above.